.nvmrc

```diff
@@ -0,0 +1 @@
+12.16.1
```

Comment:
Ja som uz nieco podobne [skusal v `packages.json`](https://github.com/priznanie-digital/priznanie-digital/blob/master/package.json#L5), ale `npm` nekontrolovalo node verziu. Ani s tyto to nekontroluje:

```
m@mac16 priznanie-digital $ node -v
v9.11.2
m@mac16 priznanie-digital $ npm install
...
added 1622 packages from 827 contributors and audited 1625 packages in 32.855s
found 0 vulnerabilities
m@mac16 priznanie-digital 
```

Neviete preco?

Reply:
Node version v package json je pre npm/yarn cize je dobre to tam mat.

`.nvmrc` je pre nvm co je najpopularnejsi node version manager. Toto v podstate zaisti ze ked som v projekte a napisem `nvm use`, rak mi to nahodi spravnu node version.

Reply:
Inac odkedy som zmenil localstorage na sessionstorage, cypress timeoutuje v ci rezime (manulane ked bezi v browsri tak vsetko je ok)

Nestretol si sa s tym?

Reply:
Aha tak. Myslel som ze sa to skontroluje aj pri `npm` / `node` comandoch.

S tym Cypress issue som sa nestretol.

Reply:
@mathio co hovoris nato ze tu dam nejaky guard aby to fungovalo zatial len v dev? Ulahci to celkom debugovanie kedze mozes reloadnut stranku pri hociktorom kroku

Reply:
Na to mame cookie: https://trello.com/c/ZCcNoMKY/51-priamy-pristup-na-stranku-cez-url

Alebo mozes pustat automatizovane testy, tak sa to otestuje rychlejsie. Radsej by som nemal taketo rozdiely medzi dev a prod. @wormyy co myslis?

Reply:
Ta cookie ale neriesi aby ostali vsetky data ktore som zatial zadal ak refreshmen nie?
Nie je to ale big deal a ten prod/dev rozdiel je valid point :)

Reply:
Nie, neriesi.

Reply:
Ja by som toto tiez zatial velmi neriesil, to neukladanie nevnimam ako nejaky velky pain point, vacsinov pouzivam testy